keyboards/thevankeyboards/caravan/rules.mk

```diff
@@ -0,0 +1,28 @@
+# MCU name
+MCU = atmega32u4
+
+
+# Bootloader selection
+# Teensy       halfkay
+# Pro Micro    caterina
+# Atmel DFU    atmel-dfu
+# LUFA DFU     lufa-dfu
+# QMK DFU      qmk-dfu
+# atmega32a    bootloadHID
+BOOTLOADER = atmel-dfu
+
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = no      # Virtual DIP switch configuration(+1000)
```

Comment:
Does this board have a physical and easily accessible reset button? If not, it may be a good idea to set this to `lite`, so that holding the key at (0,0) while plugging it in jumps to the bootloader.